src/Api/Api.php

```diff
@@ -169,7 +169,11 @@ public function call(string $path = null, string $method = 'GET', array $request
         $auth   = $this->route->attributes()['auth'] ?? true;
 
         if ($auth !== false) {
-            $this->authenticate();
+            $user = $this->authenticate();
+            
+            if (is_a($user, 'Kirby\Cms\User') === true) {
+                setlocale(LC_ALL, $user->language());
```

Comment:
I'm not sure if `setlocale()` accepts simple language codes as locales. But I also don't know how to do this in a better way.

Maybe we could check if the language code contains an underscore (like `pt_BR`). If not, we would do `$user->language() . '_' . strtoupper($user->language())`. Then again it's unclear whether we should add system fallback locales like `.ISO8859-1` or `.UTF-8` or `.UTF8`.

Reply:
I honestly do know too little about locales in PHP to chime in here.